### ISSUE TITLE
[Game]: Tom Clancy's Rainbow Six Siege

### ISSUE THREAD

**Issue by EvTardoque** [ItsLogic/Xbox-Achievement-Unlocker#277]
### Confirmation

- [X] I have verified that this is not a duplicate request.

### Game Link

https://www.trueachievements.com/game/Tom-Clancys-Rainbow-Six-Siege/achievements

### Game Platform

Console and PC

### Additional Game Information

Tom Clancy's Rainbow Six Siege Ubisoft

**Comment by pooflabber** [comment]
siege achievements are easy though

**Comment by EvTardoque** [comment]
But some are very complicated mainly because they are within the game, please put Siege

**Comment by Abro61209** [comment]
Yeah Siege definitely needs to be supported. It's buggy as hell. 

I haven't touched the game in like 7 years, but I reinstalled it because I noticed I'm missing ONE achievement. It's senseless, and I'm at 70% for it. 

I've gotten 5 flash grenade kills in the past 24 hours but it's not progressing or unlocking.

**Comment by nacliaf** [comment]
I am curious, where does siege have achievements on pc?

**Comment by Abro61209** [comment]
Siege only has achievements on PC through ubisoft connect. I found a way to unlock those achievements through a hack, but it doesn't seem like they carry over. You can't play siege on PC through Xbox. 

This is for the console version.

**Comment by nacliaf** [comment]
Exactly, and console games aren’t compatible yet
